src/components/app/Home.css

```diff
@@ -188,6 +189,16 @@
   transition: opacity 300ms ease-out, transform 300ms ease-out;
 }
 
+input[type="file"] {
```

Comment:
nit: can you add in a comment that this code comes from https://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/ ?

Reply:
Also I think we should add a specific class name for this input, for example `homeSectionUploadInput`, so that we can target this specific DOM node and this doesn't apply to all file inputs in the whole app.